### ISSUE TITLE
Capture retval parameter attribute

### ISSUE THREAD

**Issue by kennykerr** [microsoft/win32metadata#207]
Many COM interface methods include a trailing `retval` parameter that is meant to be used by languages to translate the parameter  into a return value. The metadata needs to capture this in some way, perhaps an attribute on the parameter. For example, `ISpellCheckerFactory` has an `IsSupported` method defined in the Windows SDK's `spellcheck.idl` as follows:

```C++
HRESULT IsSupported([in] LPCWSTR languageTag, [out, retval] BOOL* value);
```

And then MIDL generates:

```C++
virtual HRESULT STDMETHODCALLTYPE IsSupported( 
    /* [in] */ __RPC__in LPCWSTR languageTag,
    /* [retval][out] */ __RPC__out BOOL *value) = 0;
```

But the metadata doesn't capture this information:

```C#
unsafe HRESULT IsSupported(
    [In][Out][Const][NativeTypeInfo(UnmanagedType.LPWStr, IsNullTerminated = true)] ushort* languageTag, [Out] int* value);
```

**Comment by sotteson1** [comment]
Fixed with 10.0.19041.5-preview.3.

**Comment by kennykerr** [comment]
Awesome - much better. 😉
